### PR TITLE
Multi-machine blueprint terraform config example.

### PR DESCRIPTION
**Pull request by virajindasrao**
Multi-machine blueprint terraform config example - review changes.

Signed-off-by: Viraj Indasrao <virajindasrao@gmail.com>

### CODE REVIEW

File: example/multi-machine/main.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+provider  "vra7" {
+    username = "${var.username}"
+    password  = "${var.password}"
+    tenant = "${var.tenant}"
+    host = "${var.host}"
+}
+
+//Catalog "composite_catalog" coltains Linux, Windows and http (apache) designs.
```
Comment (markpeek) [inline example/multi-machine/main.tf]: terraform single line comments start with a # ([link](https://www.terraform.io/docs/configuration/syntax.html)). Please put a space after the comment start. And fix the typo "coltains" => "contains".

File: example/multi-machine/main.tf [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+resource "vra7_resource" "machine" {
+  count            = 1
+  catalog_name = "composite_catalog"
```
Comment (markpeek) [inline example/multi-machine/main.tf]: Is this a commonly found catalog item?

**Review by markpeek — COMMENTED** [review]
Also, didn't we want to move the other files in the example directory to a sub-folder as well?

File: example/multi-machine/main.tf (hunk 1 above)
Comment (virajindasrao) [reply example/multi-machine/main.tf]: Made necessary changes.

File: example/multi-machine/main.tf (hunk 2 above)
Comment (virajindasrao) [reply example/multi-machine/main.tf]: Name changed to multi_machine_catalog. It makes more sense now.

**Pull request closed without merging**